lib/poxa/web_hook/dispatcher.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Poxa.WebHook.Dispatcher do
+  use GenServer
+
+  alias Poxa.WebHook.EventTable
+  alias Poxa.CryptoHelper
+
+  @timeout 1500
+
+  def start_link() do
+    GenServer.start_link(__MODULE__, :ok, [])
+  end
+
+  def handle_info(:timeout, []) do
+    case EventTable.ready do
+      {_, []} -> {:noreply, [], @timeout}
+      {timestamp, events} ->
+        :ok = send_web_hook(events)
+        EventTable.clear_older(timestamp)
+        {:noreply, [], @timeout}
+    end
+  end
+
+  def init(:ok) do
+    {:ok, [], @timeout}
+  end
+
+  defp send_web_hook(events) do
```

Comment:
let's rename it to `send_web_hook!` just so it gives a hint that here's where we `send!!!`